.github/workflows/example-basic-pnpm.yml

```diff
@@ -21,13 +21,17 @@ jobs:
         # See https://github.com/pnpm/action-setup
       - name: Install pnpm
         uses: pnpm/action-setup@v4
+        # Optional when there is a `packageManager` field in the `package.json`.
+        # See https://github.com/pnpm/action-setup?tab=readme-ov-file#version
         with:
           version: 9
 
         # See https://github.com/actions/setup-node
       - name: Install Node.js
         uses: actions/setup-node@v4
         with:
+          # Note that pnpm `use-node-version` is not supported, yet
+          # See https://github.com/actions/setup-node/issues/1130
```

Comment:
https://github.com/actions/setup-node/issues/1130 is an open enhancement request

This isn't really the right place to be listing an enhancement request for an external action, when this isn't used in the example.

Reply:
My point was to link to some place that explains that there is no way to set the node version based on the config that pnpm supports. I don't see a better place where this is documented.